app/presenters/detainee_presenter.rb

```diff
@@ -0,0 +1,23 @@
+class DetaineePresenter < SimpleDelegator
+  def short_gender
+    if gender == 'male'
+      'M'
+    elsif gender == 'female'
+      'F'
+    end
+  end
+
+  def age
+    # TODO: Add validation on date_of_birth and remove presence check
```

Comment:
All the fields in the detainee must be provided, whether if they come from NOMIS or by the prison staff. This is just a remainder that we have to implement that logic sooner or later. They will ask for it.